sdks/java/io/redis/src/test/java/org/apache/beam/sdk/io/redis/RedisIOTest.java

```diff
@@ -186,6 +186,32 @@ public void testWriteReadUsingRpushMethod() throws Exception {
     Assert.assertEquals(value + newValue, String.join("", values));
   }
 
+  @Test
+  public void testWriteUsingHLLMethod() throws Exception {
+    String key = "key";
+
+    Jedis jedis =
+        RedisConnectionConfiguration.create(REDIS_HOST, embeddedRedis.getPort()).connect();
+
+    PCollection<KV<String, String>> write =
+        writePipeline.apply(
+            Create.of(
+                KV.of(key, "0"),
+                KV.of(key, "1"),
+                KV.of(key, "2"),
+                KV.of(key, "3"),
+                KV.of(key, "4"),
+                KV.of(key, "5")));
```

Comment:
Since this is a test of writing into HLL structure in Redis, could you also check adding the same values (not only uniq as it is for now)? 
For example, just add another `KV.of(key, "1")` in your input collection and check that cardinality has not been changed.

Reply:
Done